routes/home/index.js

```diff
@@ -100,14 +100,8 @@ export default () => {
             </p>
             <button
               onClick=${_ => {
-                if (window.innerWidth < 900) {
```

Comment:
I'd still like to address how the _control panel_ within each test case looks on really small screens (like mobile small)

Reply:
Currently this happens:
![Screen Shot 2019-04-02 at 17 44 10](https://user-images.githubusercontent.com/1457604/55420447-fb285a00-556e-11e9-9a78-fd89d808a289.png)

Reply:
![localhost_8080_(iPhone 6_7_8 Plus)](https://user-images.githubusercontent.com/21795/55429920-85c78400-5584-11e9-8bb7-e95047825544.png)

Reply:
iPhone 5:
![localhost_8080_(iPhone 5_SE)](https://user-images.githubusercontent.com/21795/55430012-dccd5900-5584-11e9-9f44-046029ee54d6.png)

iPhone 6/7/8 Plus:
![localhost_8080_(iPhone 6_7_8 Plus)](https://user-images.githubusercontent.com/21795/55430021-e5be2a80-5584-11e9-8b88-f501662f842a.png)

Reply:
Check the live link below for update